data/settings/src/main/java/com/google/jetpackcamera/settings/model/QuickSettings.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jetpackcamera.settings.model
+
+data class QuickSettings(
```

Comment:
A couple nits:
1. use camel case instead of snake case
2. you can set the default value to be false or if you need to know if it's a default instance then you create a `const val DEFAULT_QUICK_SETTINGS = QuickSettings()`

```
data class QuickSettings(val defaultFrontCamera: Boolean = false)
const val DEFAULT_QUICK_SETTINGS = QuickSettings()
```